landoapi/phabricator.py

```diff
@@ -198,6 +198,14 @@ def call_conduit(self, method, **kwargs):
 
         data = {"output": "json", "params": json.dumps(kwargs)}
 
+        extra_data = {
+            "params": kwargs.copy(),
+            "method": method,
+            "api_url": self.api_url,
+        }
+        del extra_data["params"]["__conduit__"]  # Sanitize the api token.
+        logger.debug("call to conduit", extra=extra_data)
```

Comment:
If you want to keep all the logging call objects temporary in a single call you could go with something like:
```
logger.debug(
    "call to conduit",
    extra={
        "params": {k: kwargs[k] for k in kwargs.keys() - {"__conduit__"}},
        "method": method,
        "api_url": self.api_url,
    }
)
```